src/components/CaptureDetailDialog.js

```diff
@@ -261,40 +296,42 @@ function CaptureDetailDialog(props) {
   }
 
   return (
-    <Dialog
-      open={open}
-      TransitionComponent={TransitionComponent}
-      onClose={handleClose}
-      maxWidth="xl"
-    >
-      <DialogContent>
-        <Grid container spacing={4} wrap="nowrap">
-          <Grid item>
-            <OptimizedImage
-              src={renderCapture.imageUrl}
-              width={640}
-              style={{ maxWidth: '100%' }}
-              fixed
+    <React.Fragment>
+      <Dialog
+        open={open}
+        TransitionComponent={TransitionComponent}
+        onClose={handleClose}
+        style={{ width: screenWidth - 340 }}
+        BackdropProps={{
+          classes: {
+            root: classes.dialog,
+          },
+        }}
+      >
+        <OptimizedImage
+          src={renderCapture.imageUrl}
+          width={320}
```

Comment:
I think we can make the capture image bigger now there's a bit more room for it.
We have three options, any of which would work well:

1. Dynamically calculate what size of image would fit in the space and request that using the `OptimizedImage` component
2. Drop `OptimizedImage` for the full-size image and just shrink the original image to fill the space
3. Set the `OptimizedImage` width to `600`